python/kfserving/README.md

```diff
@@ -50,7 +50,7 @@ KFServing supports the following storage providers:
 
 ### Getting Started
 
-KFServing's python client provides many APIs for operations of KFServing, such as creating, patching and deleting. See the [Sample for KFServing Python SDK Client](../../docs/samples/client/kfserving_sdk_sample.ipynb) to get started.
+KFServing's python client interacts with KFServing APIs for exeucting operations on a remote KFServing server instance, such as creating, patching and deleting of a KFService instance. See the [Sample for KFServing Python SDK Client](../../docs/samples/client/kfserving_sdk_sample.ipynb) to get started.
```

Comment:
typo
```suggestion
KFServing's python client interacts with KFServing APIs for executing operations on a remote KFServing server instance, such as creating, patching and deleting of a KFService instance. See the [Sample for KFServing Python SDK Client](../../docs/samples/client/kfserving_sdk_sample.ipynb) to get started.
```

Reply:
instead of remote server instance maybe say on a KFServing deployment or cluster?

Reply:
thanks @rakelkar - had 'cluster' there before. Will update

Reply:
Done